.circleci/config.yml

```diff
@@ -206,18 +206,18 @@ workflows:
       - run:
           matrix:
             parameters:
-              python_version: [ "3.7", "3.8", "3.9", "3.10" ]
+              python_version: [ "3.8", "3.9", "3.10" ]
```

Comment:
Should we be adding 3.11 here at some point?

Reply:
Yes, we will need to update CI to GitHub actions for it as well.